examples/cliffordexamples/destabilizer.txt

```diff
@@ -0,0 +1,8 @@
+Destabilizer = ['+IZ', '+XI']
+Destabilizer = ['-XX', '+XI']
+Destabilizer = ['+XX', '+XI']
+Destabilizer = ['+XY', '+XI']
+Destabilizer = ['+XY', '+ZZ']
+Destabilizer = ['+XX', '+ZZ']
+Destabilizer = ['-XX', '+ZZ']
+Destabilizer = ['+XX', '-ZZ']
```

Comment:
Is this IBM's format?

Reply:
Yes, this is IBMs format for stabilizer tables